packages/flutter_migrate/lib/src/flutter_project_metadata.dart

```diff
@@ -252,7 +252,7 @@ class MigrateConfig {
   /// Parses the project for all supported platforms and populates the [MigrateConfig]
   /// to reflect the project.
   void populate({
-    List<SupportedPlatform>? platforms,
+    List<SupportedPlatform?>? platforms,
```

Comment:
i guess same question here, is null a valid platform?

Reply:
See discussion below, changed to use a new `FlutterProjectComponent` enum that includes root as a value.